tensorflow/core/graph/mkl_layout_pass_test.cc

```diff
@@ -1037,6 +1037,321 @@ TEST_F(MklLayoutPassTest, NodeMerge_TransposeConv2DTranspose_Negative) {
       "_1:control;Transpose1->Relu;Transpose1:control->DMT/_2:control");
 }
 
+TEST_F(MklLayoutPassTest, NodeMerge_TransposeConv3DTranspose_Positive) {
+  InitGraph(
```

Comment:
We don't have any format tool for this, even minor changes need manually fix...

I don't think we can format my 2 cases as the example you provide. It's true the example you provide is neat and graceful, because it's a relatively simple case that all "attr" can be spanned in a single line. My cases are more complicated that "attr" would span a few lines (i.e. tensor_content), so I'd rather keep it as the prototxt format (attributes spans in lines), which is a standard output protobuf recommend.

Reply:
But you're right, we should make the format friendly to read. I've made some changes to adapt consistent indentations in my cases. Currently the format is strictly the standard "prototxt" format, which is easy to read I think.

Reply:
Got it. Thank you for clarifying and for making the changes!
Would you mind changing the formatting here (multiple one-line strings) to a multi-line string like the rest of the file?

Reply:
My last question hasn't been addressed.